hueplusplus/include/IHttpHandler.h

```diff
@@ -183,8 +183,8 @@ class IHttpHandler
 		std::unique_ptr<Json::CharReader> reader = std::unique_ptr<Json::CharReader>(builder.newCharReader());
 		if (!reader->parse(response.c_str(), response.c_str() + response.length(), &result, &error))
 		{
-			std::cout << "Error while parsing JSON in function GETJson() of linHttpHandler: " << error << std::endl;
-			throw(std::runtime_error("Error while parsing JSON in function GETJson() of linHttpHandler"));
+			std::cout << "IHttpHandler: Error while parsing JSON in function GETJson(): " << error << std::endl;
```

Comment:
Error message printed to std::cout, but above it was printed to std::cerr. Consistently use std::cerr

Reply:
Changed in 88d61e3 and 9ce3e58